File: db4oj.tests/src/com/db4o/db4ounit/common/cs/AllTests.java
/* Copyright (C) 2004 - 20067 Versant Inc. http://www.db4o.com */

package com.db4o.db4ounit.common.cs;

import db4ounit.extensions.*;

public class AllTests extends Db4oTestSuite {

	public static void main(String[] args) {
		new AllTests().runAll();
    }
	
	protected Class[] testCases() {
		return new Class[] {
		        CallConstructorsConfigTestCase.class,
		        ClientDisconnectTestCase.class,
	            ClientTimeOutTestCase.class,
	            ClientTransactionHandleTestCase.class,
	            ClientTransactionPoolTestCase.class,
	            CloseServerBeforeClientTestCase.class,
	            CsCascadedDeleteReaddChildReferenceTestCase.class,
	            CsDeleteReaddTestCase.class,
	            IsAliveTestCase.class,
	            NoTestConstructorsQEStringCmpTestCase.class,
	            ObjectServerTestCase.class,
	            PrimitiveMessageTestCase.class,
	            QueryConsistencyTestCase.class,
	            SendMessageToClientTestCase.class,
	            ServerClosedTestCase.class,
	            ServerPortUsedTestCase.class,
	            ServerRevokeAccessTestCase.class,
	            ServerTimeoutTestCase.class,
	            ServerToClientTestCase.class,
	            SetSemaphoreTestCase.class,
	            SwitchingFilesFromClientTestCase.class,
	            SwitchingFilesFromMultipleClientsTestCase.class,
	            SwitchingToFileWithDifferentClassesTestCase.class,
	            CsSchemaUpdateTestCase.class,
		};
	}
	
}

File: db4on/test/src/com/db4o/db4ounit/common/defragment/AllTests.cs
namespace com.db4o.db4ounit.common.defragment
{
	public class AllTests : Db4oUnit.Extensions.Db4oTestSuite
	{
		public static void Main(string[] args)
		{
			new com.db4o.db4ounit.common.defragment.AllTests().RunSolo();
		}

		protected override System.Type[] TestCases()
		{
			return new System.Type[] { typeof(com.db4o.db4ounit.common.defragment.SlotDefragmentTestCase)
				 };
		}
	}
}
